.github/workflows/build.yml

```diff
@@ -70,3 +82,7 @@ jobs:
           push: ${{ github.event_name != 'pull_request' && github.repository == 'photoview/photoview' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
+          build-args: |
+            BUILD_DATE=${{ steps.args.outputs.BUILD_DATE }}
+            VERSION=${{ steps.args.outputs.VERSION }}
```

Comment:
Please replace this line with the next one:
`VERSION=${{ github.ref_name }}`
It should be resolved to the correct version for all the supported cases: 
- for builds in `master` branch it will return the branch name, which is the same as the build tag, and the same as the version, we currently use for 'edge' builds
- for builds in tagged commits it will return the tag name, which is what we want to be set as our version as well.
- for PRs it will return the PR ID, but the image is not pushed anywhere, so we don't care)

Reply:
Then `VERSION` has a prefix `v`, which doesn't currently.

Reply:
As it is only shown to users, I don't see an issue here.
However, as we still need the `run` step to handle build time, It is also fine if you'd prefer to keep the code for tag name parsing

Reply:
Done with `github.ref_name`.